portfolio/src/main/webapp/home.html

```diff
@@ -9,7 +9,7 @@
 	<script src="script.js"></script>
 </head>
 
-<body onload="parseJSON()">
```

Comment:
Is this parseJSON function now unused? Maybe it should be deleted from script.js then?

Reply:
Gotcha